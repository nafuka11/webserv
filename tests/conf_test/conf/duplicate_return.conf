server {
    location /302 {
        return 302 /302.html;
        return 301 /301.html;
    }
}