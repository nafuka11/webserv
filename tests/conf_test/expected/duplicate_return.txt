webserv: duplicate "return" directive in ./conf/duplicate_return.conf:4
